plugin.js

```diff
@@ -352,10 +352,17 @@ function processWorkletFunction(t, fun, fileName) {
   const funString = buildWorkletString(t, fun, variables, functionName);
   const workletHash = hash(funString);
 
-  const loc = fun?.node?.loc?.start;
-  if (loc) {
-    const { line, column } = loc;
-    fileName = `${fileName} (${line}:${column})`;
+  try {
+    // try to append location in file
+    const loc = fun.node.loc.start;
+    if (loc) {
+      const { line, column } = loc;
+      fileName = `${fileName} (${line}:${column})`;
+    }
+  } catch (e) {
+    if (!(e instanceof TypeError)) {
+      throw e;
+    }
```

Comment:
I'd rather wrap in try only access to this object:
```
let loc;
try {
	const loc = fun.node.loc.start;
} catch (e) {
    if (!(e instanceof TypeError)) {
      throw e;
}
```

or use old-fashioned method:
```
const loc = fun && fun.node && fun.node.loc && fun.node.loc.start
```
which will allow removing try-catch entirely.

Reply:
I was considering all the options you mentioned. Picked the wider try catch because it just seemed to be the clearest one to me. Doesn't really matter to me, I can change it or leave it as it is.

Reply:
ok I updated the code